conan/tools/apple/apple.py

```diff
@@ -18,19 +18,21 @@ def to_apple_arch(arch, default=None):
             'armv7k': 'armv7k'}.get(arch, default)
 
 
-def get_apple_sdk_name(conanfile):
+def get_apple_sdk_fullname(conanfile):
```

Comment:
changed the name of this function to remark that it returns sdk_name with version if exists